src/app/room/room.component.spec.ts

```diff
@@ -18,17 +20,44 @@ describe('RoomComponent', () => {
 
   beforeEach(() => {
     formBuilder = {} as FormBuilder;
-    router = {} as Router;
-    route = {} as ActivatedRoute;
+    router = { navigate: _ => {} } as Router;
+    route = { snapshot: { params: {} } } as ActivatedRoute;
     const document = { valueChanges: () => of({}) };
     playerService = { getCurrentPlayerDocument: () => of(document) } as FirebasePlayerService;
-    playerCreationService = {} as PlayerCreationService;
-    store = { select: (_: string) => {} } as Store;
+    playerCreationService = { createPlayer: _ => {} } as PlayerCreationService;
+    store = {
+      select: (_: string) => of({}),
+      dispatch: _ => {}
+    } as Store;
 
     component = new RoomComponent(formBuilder, router, playerCreationService, playerService, route, store);
   });
 
-  it('should create', () => {
-    expect(component).toBeTruthy();
+  it('creates new player OnInit if there is none in the store', () => {
+    spyOn(store, 'select').and.returnValue(of({}));
+    spyOn(playerCreationService, 'createPlayer');
+    route.snapshot.params[constants.routeParams.id] = 'roomId';
+
+    component.ngOnInit();
+
+    expect(playerCreationService.createPlayer).toHaveBeenCalledWith('roomId');
```

Comment:
Do I see extra line here?
<img width="219" alt="image" src="https://user-images.githubusercontent.com/17585880/200166870-8fdaea39-02ff-4ba4-a502-6a87eca2830f.png">

Reply:
I think that is just rendered strangely on your mac:

![image](https://user-images.githubusercontent.com/17160319/200180264-150064e2-cf08-4b06-bdd3-4a516e86fbe0.png)